distribution/scenes/textures/patterns/agate.pov

```diff
@@ -2,35 +2,38 @@
 // To view a copy of this license, visit http://creativecommons.org/licenses/by/3.0/
 // or send a letter to Creative Commons, 444 Castro Street, Suite 900, Mountain View,
 // California, 94041, USA.
-
-// Persistence Of Vision raytracer sample file.
-// Agate pattern example
 //
-// -w320 -h240
+// Persistence Of Vision Ray Tracer ('POV-Ray') sample file.
+//
+// Agate pattern example agate.pov.
+//
```

Comment:
I thought we might keep it around for thumbnails. Of course they should be consistent in aspect ratio with the larger image.